src/govee_ble/parser.py

```diff
@@ -26,6 +26,9 @@
 PACKED_hhhhh = struct.Struct(">hhhhh")
 
 
+UNAVAILABLE = "unavailable"
```

Comment:
Is there an existing constant available?

Reply:
We would need to implement something in sensor_state_data to be able to tell Home Assistant it's in an error state and mark it unavailable.  Right now there it's typed to only accept a float value so a string will break the typing checks.

Reply:
[update_predefined_sensor](https://github.com/Bluetooth-Devices/sensor-state-data/blob/eca174a97b36e8229dad9fb415fa37a50c4c3024/src/sensor_state_data/data.py#L216) and [update_sensor](https://github.com/Bluetooth-Devices/sensor-state-data/blob/eca174a97b36e8229dad9fb415fa37a50c4c3024/src/sensor_state_data/data.py#L265) are typed to accept a str, but indeed using a string here is wrong.
[        native_value: None | str | int | float | date | datetime | Decimal,](https://github.com/Bluetooth-Devices/sensor-state-data/blob/eca174a97b36e8229dad9fb415fa37a50c4c3024/src/sensor_state_data/data.py#L219)

`None` did not work for me: resulted in Unknown instead of unavailable

Reply:
Home Assistant requires the `available` property to be set if you want to make the sensor unavailable. Its not allowed to set `unavailable` as a string.

Reply:
https://developers.home-assistant.io/docs/core/entity?_highlight=entity#generic-properties